src/pages/VideoPage/components/VideoSection/VideoSection.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import { AiOutlineLike } from "react-icons/ai";
+import { MdOutlineWatchLater, MdPlaylistPlay } from "react-icons/md";
+import { images } from "../../../../assets";
+
+import "./VideoSection.css";
+
+const video = {
+  _id: "31UVvKEYO6k",
+  thumbnail: images.boat,
+  title:
+    "How Aman Gupta's MARKETING STRATEGY turned Boat into a 1500CR Company : Business case study",
+  description:
+    "Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus, labore. Sint eum amet, inventore itaque, asperiores earum assumenda accusamus illo a similique dolor hic vero non sunt deleniti iste aliquam suscipit culpa provident. Temporibus natus deleniti sequi perspiciatis totam recusandae, laboriosam odio voluptatibus, quam id, odit cum animi pariatur facilis.",
+  creator: "Think School",
+  creatorImg: images.thinkSchool,
+  category: ["case study"],
+  views: "971,345",
+  label: [""],
+  date: "Jan 28, 2022",
+};
```

Comment:
it would be better if data come from backend

Reply:
Functionality yet to be implemented